### PR TITLE
chore: release version 2.5.1 package

### PR DESCRIPTION
**Pull request by dnshi**
# Summary
- Bump up version to v2.5.1
- Update `package.json` file
- Generate a new `.unitypackage` file

### CODE REVIEW

File: ProjectSettings/MemorySettings.asset [hunk 1]
```diff
@@ -0,0 +1,35 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!387306366 &1
+MemorySettings:
```
Comment (qingzhuozhen) [inline ProjectSettings/MemorySettings.asset:4]: Do we need this file?

File: ProjectSettings/ProjectVersion.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.18f1
-m_EditorVersionWithRevision: 2019.4.18f1 (3310a4d4f880)
+m_EditorVersion: 2021.3.3f1
```
Comment (qingzhuozhen) [inline ProjectSettings/ProjectVersion.txt:1]: I guess we will use the new version in the future

**Review by yuhao900914 — APPROVED** [review]

**Review by qingzhuozhen — APPROVED** [review]

File: ProjectSettings/MemorySettings.asset (hunk 1 above)
Comment (dnshi) [reply ProjectSettings/MemorySettings.asset:4]: We might not need it but it is generated by Unity. It is not included into the `.unitypackage`.

File: ProjectSettings/ProjectVersion.txt (hunk 2 above)
Comment (dnshi) [reply ProjectSettings/ProjectVersion.txt:1]: Right. `2021.3.3f1` is current active LTS. We will update the new version when the next LTS is active.

**Review by qingzhuozhen — APPROVED** [review]

**Pull request merged**